yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.ts

```diff
@@ -602,13 +604,10 @@ export class PXEOracleInterface implements ExecutionDataProvider {
         throw new Error(`Could not find tx effect for tx hash ${scopedLog.txHash}`);
       }
 
-      // We don't perform tagging in Noir and for this reason we pass the log into `process_log` without the tag.
-      const logFieldsWithoutTag = scopedLog.log.toFields().slice(1);
```

Comment:
Needed the tag info to determine whether a log has been processed in the private_event_data_provider.ts